src/Symfony/Component/Security/Core/Tests/Authorization/AccessDecisionManagerTest.php

```diff
@@ -102,7 +102,7 @@ public function getStrategiesWith2RolesTests()
     protected function getVoterFor2Roles($token, $vote1, $vote2)
     {
         $voter = $this->getMock('Symfony\Component\Security\Core\Authorization\Voter\VoterInterface');
-        $voter->expects($this->exactly(2))
+        $voter->expects($this->any())
```

Comment:
`exactly(2)` is wrong, and in fact, we don't care about the count, but only about the final result of the vote